docs/Pitfalls.asciidoc

```diff
@@ -53,3 +53,25 @@ impacted by snapshot creation. Loading VM snapshots can also have an impact on
 SUT behavior as the execution of the first step after loading a snapshot might
 be delayed. This can lead to problems if the executed tests do not foresee an
 appropriate timeout margin.
+
+== DB migration from sqlite to postgreSQL
+As a first step  to start using postgreSQ, please, configure postgreSQL database
+according to the
+link:Contributing.asciidoc#how-to-setup-postgresql-to-test-locally-with-production-data[postgreSQL setup guide]
```

Comment:
When https://github.com/os-autoinst/openQA/pull/1538 is merged, this should better link to that guide. The referenced guide had more the setup for developing openQA itself in mind.

Reply:
Right, I'll fix the reference after that.